addons/skin.estouchy/language/resource.language.en_gb/strings.po

```diff
@@ -179,7 +179,7 @@ msgstr ""
 #empty strings from id 31304 to 31308
 
 msgctxt "#31309"
-msgid "Memory Used:"
+msgid "System Memory Used:"
```

Comment:
Sure and it was already there (and every other word in the language file also has it)